overlay/tun_linux.go

```diff
@@ -350,6 +380,7 @@ func (t *tun) Activate() error {
 	if err = ioctl(t.ioctlFd, unix.SIOCSIFFLAGS, uintptr(unsafe.Pointer(&ifrf))); err != nil {
 		return fmt.Errorf("failed to run tun device: %s", err)
 	}
+	//todo do we want to keep the link-local address?
```

Comment:
Ideally no but in bsd land I could not find a way to avoid the auto configuration so we'd have to remove it after adding the first v6 address to the network device.

Reply:
Yep that's what I ended up doing